app/assets/stylesheets/ama_layout/layout/helper-classes.scss

```diff
@@ -235,6 +242,7 @@
   line-height: 1.3rem !important;
 }
 
+
```

Comment:
Hm, in the actual file line 245 is showing up as a } and 426 is just the blank line we usually have at the bottom of every file.